WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -388,6 +391,10 @@ private void refreshBlogDetails(@Nullable final Blog blog) {
         } else {
             mPlanContainer.setVisibility(View.GONE);
         }
+
+        // toggle people row visibility
+        boolean canVisitPeople = blog.checkCapability(Capability.LIST_USERS);
```

Comment:
This is relatively nitpicking but, I'd suggest naming this helper variable to something closer to the capability, like "canListPeople" or even "canListUsers" in order to not introduce a new verb ("visit") in our vocabulary around this feature. `LIST_USERS` is pretty accurate anyway and doesn't ask for an interpretation like other capabilities do. Cheers!